### PR TITLE
Bug 1188103 - Update test_video_playback for new MSE

### PR DESCRIPTION
**Pull request by sydvicious**
- Detect stalls by tracking currentTime vs. wall time
- Handle partial videos in video_puppeteer now, removing awkward exception handling in run_playback
- Only run partial videos for 1 minute instead of 2.

@mjzffr , please review. Thanks.

### CODE REVIEW

**Comment by sydvicious** [comment]
OK, obviously I need to rebase. Having trouble doing that tonight; maybe you can help me with it on Wednesday. Please review anyway; I am sure that there will be changes.

File: firefox_media_tests/playback/test_video_playback.py [hunk 1]
```diff
@@ -28,12 +28,15 @@ def tearDown(self):
     def test_playback_starts(self):
         with self.marionette.using_context('content'):
             for url in self.test_urls:
+                print url
```
Comment (mjzffr) [inline firefox_media_tests/playback/test_video_playback.py:31]: The url is already logged by marionette in the `VideoPuppeteer` constructor. Is this left over from debugging?

**Comment by mjzffr** [comment]
Hey @sydvicious, thanks for the PR -- I'll be able to take a close look at this tomorrow.

As agreed yesterday, let's try tracking this in https://bugzilla.mozilla.org/show_bug.cgi?id=1188103 -- could you attach this PR to the bug and flag me for review there? Feel free to change the bug summary, etc, too. :)

File: firefox_media_tests/playback/test_video_playback.py (hunk 1 above)
Comment (sydvicious) [inline firefox_media_tests/playback/test_video_playback.py:31]: I want to see it in the console as a progress indicator.

**Comment by sydvicious** [comment]
When I get the correct branches setup and the commits squished, yes.

**Comment by sydvicious** [comment]
Closing this and re-submitting with a properly-based branch.

**Pull request closed without merging**